### ISSUE TITLE
prepInputs treats Google Spreadsheets urls as zip files

### ISSUE THREAD

**Issue by achubaty** [PredictiveEcology/reproducible#287]
When passing a url for a Google Sheets file, `prepInputs()` is treating it like a `.zip` file instead of e.g., a `.csv` and errors:

```r
userDist <- prepInputs(
  url = "https://docs.google.com/spreadsheets/d/1fOikb83aOuLlFYIn6pjmC7Jydjcy77TH", ## userDist.csv
  destinationPath = tempdir(),
  fun = "data.table::fread"
)
```

```r
Feb27 11:12:20 simInit: CBM_dt Running preProcess
Feb27 11:12:20 simInit: CBM_dt targetFile was not supplied; guessed and will try /tmp/RtmpJqz2xL/1fOikb83aOuLlFYIn6pjmC7Jydjcy77TH. If this is incorrect, please supply targetFile
Feb27 11:12:20 simInit: CBM_dt Preparing: 1fOikb83aOuLlFYIn6pjmC7Jydjcy77TH
Feb27 11:12:20 simInit: CBM_dt Checking local files...
Feb27 11:12:20 simInit: CBM_dt Finished checking local files.
Feb27 11:12:20 simInit: CBM_dt   Downloading https://docs.google.com/spreadsheets/d/1fOikb83aOuLlFYIn6pjmC7Jydjcy77TH ...
  |======================================================================================================================================| 100%
  |======================================================================================================================================| 100%
Downloading: 53 kB     Feb27 11:12:21 simInit: CBM_dt Hardlinked version of file(s) created at: /tmp/RtmpJqz2xL/1fOikb83aOuLlFYIn6pjmC7Jydjcy77TH, which point(s) to /tmp/RtmpJqz2xL/reproducible/5y1dXR/1fOikb83aOuLlFYIn6pjmC7Jydjcy77TH; no copy/copies made.
Feb27 11:12:21 simInit: CBM_dt Checking local files...
Feb27 11:12:21 simInit: CBM_dt Finished checking local files.
Feb27 11:12:21 simInit: CBM_dt Downloaded file has no extension: neither archive nor targetFile are provided. prepInputs will try accessing the file type.
Feb27 11:12:21 simInit: CBM_dt The file was not recognized by prepInputs. Will assume the file is an archive and add '.zip' extension. If this is incorrect or return error, please supply archive or targetFile
Error in funWArgs$fun(archive[1], list = TRUE) : 
  zip file '/tmp/RtmpJqz2xL/1fOikb83aOuLlFYIn6pjmC7Jydjcy77TH.zip' cannot be opened
```

**Comment by achubaty** [comment]
THe following works:

```r
drive_download(as_id("1fOikb83aOuLlFYIn6pjmC7Jydjcy77TH"),
               path = file.path(tempdir(), "userDist.csv"),
               type = "spreadsheet") ## need to spcify correct type
```

**Comment by eliotmcintire** [comment]
I tried your code and followed the message. The solution suggested in the message works:
```
userDist <- prepInputs(
    url = "https://docs.google.com/spreadsheets/d/1fOikb83aOuLlFYIn6pjmC7Jydjcy77TH", ## 
    targetFile = "userDist.csv", # <---------------------------------------------------------------- specify targeFile
    destinationPath = tempdir(),
    fun = "data.table::fread"
)
```

**Comment by eliotmcintire** [comment]
Closes with 12a3fe440057c9c5e6c547925402983615a2197c
